src/client/dfuse/SConscript

```diff
@@ -116,6 +116,19 @@ def check_struct_member(env, text, struct, member):
     env.Result(rc)
     return rc
 
+def configure_fuse(cenv):
+    """Configure for specific version of fuse"""
+    if GetOption('help'):
+        return
+
+    check = Configure(cenv, custom_tests={'CheckStructMember': check_struct_member})
+
+    if check.CheckStructMember('#include <fuse3/fuse.h>',
```

Comment:
(pylint-no-member) Instance of 'Configure' has no 'CheckStructMember' member

Reply:
Now might be a good time to resolve this now we've reconciled scons_local and daos.

Reply:
we could add it to fake_scons too if that's what you mean?